scripts/release/rpm/test_rpm_in_docker.sh

```diff
@@ -15,7 +15,7 @@ time az self-test
 time az --version
 
 cd /azure-cli/
-python -m pip install --upgrade pip setuptools
+python -m pip install setuptools
```

Comment:
I suggest we won't make the change for other Linux RPM packages.

Reply:
Reverted, other rpm packages are not affected.
Not sure why azurelinux3 make such weird change.